docs/templates/topindex.html

```diff
@@ -64,7 +39,27 @@ <h1>MicroPython documentation</h1>
     </td>
   </tr></table>
 
+  <p><strong>Reference and tutorials for specific microcontrollers:</strong></p>
```

Comment:
"microcontrollers" is going to be out of place as soon as an OS or RTOS entries are added. Glossary defines term [MicroPython port](https://micropython.org/resources/docs/unified-docs/reference/glossary.html#term-micropython-port), which is used throughout the documentation and should apparently be used here too.

Reply:
I wanted to emphasise the microcontroller aspect, and indeed all these specific platforms are microcontrollers.  They have pinouts and everything.

Reply:
But there're a lot to write about Unix port for example. The reason it wasn't done before because of the mis-structure of documentation. And now even with this change, the documentation is not unifying for all ports, having bias towards "microcontrollers" (which amounts to not-inviting situation for OS/RTOS ports).

Reply:
Ok, the word "platform" is used now, so as not to introduce too much jargon on the front page.